<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wifi_cracking</class>
 <widget class="QDialog" name="wifi_cracking">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>431</width>
	<height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
	<layout class="QVBoxLayout" name="verticalLayout">
	 <item>
	  <widget class="QLineEdit" name="cracking_text">
	   <property name="enabled">
		<bool>false</bool>
	   </property>
	   <property name="text">
		<string>If WiFi scanning is finished, you must select a WiFi name.</string>
	   </property>
	  </widget>
	 </item>
	 <item>
	  <layout class="QHBoxLayout" name="horizontalLayout">
	   <item>
		<widget class="QPushButton" name="StartBtn">
		 <property name="text">
		  <string>Start</string>
		 </property>
		</widget>
	   </item>
	   <item>
		<widget class="QPushButton" name="StopBtn">
		 <property name="text">
		  <string>Stop</string>
		 </property>
		</widget>
	   </item>
	   <item>
		<widget class="QPushButton" name="HelpBtn">
		 <property name="text">
		  <string>Help</string>
		 </property>
		</widget>
	   </item>
	  </layout>
	 </item>
	 <item>
	  <widget class="QListView" name="listView"/>
	 </item>
	 <item>
	  <layout class="QHBoxLayout" name="horizontalLayout_2">
	   <item>
		<spacer name="horizontalSpacer">
		 <property name="orientation">
		  <enum>Qt::Horizontal</enum>
		 </property>
		 <property name="sizeHint" stdset="0">
		  <size>
		   <width>40</width>
		   <height>20</height>
		  </size>
		 </property>
		</spacer>
	   </item>
	   <item>
		<widget class="QPushButton" name="SelectBtn">
		 <property name="text">
		  <string>Select</string>
		 </property>
		</widget>
	   </item>
	   <item>
		<widget class="QPushButton" name="CancelBtn">
		 <property name="text">
		  <string>Cancel</string>
		 </property>
		</widget>
	   </item>
	  </layout>
	 </item>
	</layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
